### PR TITLE
feat: adjust drop folder directory structure to prepare for electron release feature

### PR DESCRIPTION
**Pull request by Shobhit1**
#### Description of changes

This PR is to make sure that the drop folder is divided properly into `extension` and `electron` folders. 
I noticed that previously, `electron` lived under `drop` and had a `extension` subfolder, which was wrong, so figured that this might be a better structure.
Open to feedback.

Now, the following structure is in place in the drop folder:

```bash
├── drop
│   ├── extension
│   │   ├── canary
│   │   │   ├── product
│   │   ├── dev
│   │   │   ├── product
│   │   ├── insider
│   │   │   ├── product
│   │   ├── playground
│   │   │   ├── product
│   │   ├── preview
│   │   │   ├── product
│   │   ├── production
│   │   │   ├── product
│   ├── electron
│   │   ├── electron
│   │   │   ├── product
└── .gitignore
```

Accompanying changes are to make sure that our code and readme and other references are updated with current directory structure.

#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [x] Updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). Check workflow guide at: `<rootDir>/docs/workflow.md`
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

**Comment by jalkire** [comment]
You may need to update [electron-builder.yml](https://github.com/microsoft/accessibility-insights-web/blob/master/electron-builder.yml) with the new structure as well

File: AppXManifest.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License.
   <Properties>
     <DisplayName>Accessibility Insights for Web</DisplayName>
     <PublisherDisplayName>Microsoft Corporation</PublisherDisplayName>
-    <Logo>extension\icons\brand\blue-on-white\brand-blue-on-white-150px.png</Logo>
```
Comment (smoralesd) [inline AppXManifest.xml]: I understand that this file has to do with edge webstore, do we know if there is any proper way to test it locally?
@peterdur may have some ideas.

File: Gruntfile.js [hunk 2]
```diff
@@ -155,11 +155,15 @@ module.exports = function(grunt) {
     const targetNames = Object.keys(targets);
     const releaseTargets = Object.keys(targets).filter(t => targets[t].release);
     targetNames.forEach(targetName => {
-        const dropPath = path.join('drop', targetName);
-        const dropExtensionPath = path.join(dropPath, 'extension');
-
         const { config, bundleFolder } = targets[targetName];
 
+        const { options } = config;
+        const { productCategory } = options;
```
Comment (smoralesd) [inline Gruntfile.js]: why not:
```javascript
const { productCategory } = config.options;
```
???

File: targets.config.js [hunk 3]
```diff
@@ -104,8 +109,9 @@ module.exports = {
             options: {
                 ...publicOptions,
                 ...icons.production,
-                extensionFullName: 'Accessibility Insights for Web - Preview',
+                fullName: 'Accessibility Insights for Web - Preview',
```
Comment (smoralesd) [inline targets.config.js]: I guess we should remove **Preview** since we are not actually publishing it but I can see that being a different PR

Comment (Shobhit1) [reply targets.config.js]: Yup. I can take care of that after this is merged!

**Comment by Shobhit1** [comment]
Closing and reopening to trigger build!

**Comment by Shobhit1** [comment]
The end-to-end failure is a know flaky test with linux on `fonts` build.

File: AppXManifest.xml (hunk 1 above)
Comment (smoralesd) [reply AppXManifest.xml]: @Shobhit1 can you follow up with @peterdur and see if there is anything we may want to test further before closing this PR?

**Pull request merged**